Cargo.toml

```
@@ -14,7 +14,9 @@ members = [
     "crates/configuration",
     "crates/universal-sierra-compiler-api",
     "crates/docs",
-    "crates/debugging"
+    "crates/debugging",
+    "crates/testing",
+    "crates/testing/packages_validation",
```

Can't we do something along the lines of this?

```suggestion
    "crates/testing/*"
```

---

I don't think so. Note that above we do the same:

https://github.com/foundry-rs/starknet-foundry/blob/master/Cargo.toml#L9-L10